platform/MCU/nRF52840/inc/platform_config.h

```diff
@@ -37,7 +37,8 @@
     PLATFORM_ID == PLATFORM_ASOM || \
     PLATFORM_ID == PLATFORM_BSOM || \
     PLATFORM_ID == PLATFORM_B5SOM || \
-    PLATFORM_ID == PLATFORM_TRACKER
+    PLATFORM_ID == PLATFORM_TRACKER || \
+    PLATFORM_ID == PLATFORM_ESOMX
```

Comment:
If the E404X is gping to adopt an 8MB external flash, we need to modify line 52.

Reply:
I think there is still discussion about if E404X will support only 4mb filesystem or not, regardless of flash part. I updated `flash_mal` to support 8mb for now

Reply:
E404X will support 8mb flash, and expose 4mb to user as FS, like tracker and b5som here.